.env.example

```diff
@@ -3,3 +3,8 @@ FACEBOOK_APP_ID=
 FACEBOOK_APP_SECRET=
 REMOTE_DATABASE_URL=
 MAPBOX_TOKEN=
+
+# If you set up postgres via Docker as per the README, you should set these
+POSTGRES_HOST=localhost
+POSTGRES_USER=postgres
+PGPASSWORD=foobar
```

Comment:
add port too?

Reply:
We don't read the port environment variable: https://github.com/geeksforsocialchange/PlaceCal/blob/1370_1372_update-readme/config/database.yml#L17